apps/music/js/metadata/core.js

```diff
@@ -96,10 +96,10 @@ var AudioMetadata = (function() {
     }
 
     // Read the start of the file, figure out what kind it is, and call
-    // the appropriate parser.  Start off with an 64kb chunk of data.
+    // the appropriate parser.  Start off with an 256kb chunk of data.
     // If the metadata is in that initial chunk we won't have to read again.
     return new Promise(function(resolve, reject) {
-      var headersize = Math.min(64 * 1024, blob.size);
+      var headersize = Math.min(256 * 1024, blob.size);
```

Comment:
I haven't fixed this yet.
The problem is that BlobView doesn't allow loading more if needed. I found that 256kb was a decent compromise between more data and not too much for what we want to do. It is not perfect, far from it.